PhysicsTools/Heppy/python/analyzers/objects/TauAnalyzer.py

```diff
@@ -81,14 +81,16 @@ def id6(tau,X):
                     loose, medium, tight, very tight, very very tight)"""
                 return id5(tau, X) + tau.tauID(X%"VVTight")
 
-            tau.idMVA = id6(tau, "by%sIsolationMVArun2v1DBoldDMwLT")
-            tau.idMVANewDM = id6(tau, "by%sIsolationMVArun2v1DBnewDMwLT")
+            tau.idMVArun2 = id6(tau, "by%sIsolationMVArun2v1DBoldDMwLT")
+            tau.idMVArun2dR03 = id6(tau, "by%sIsolationMVArun2v1DBdR03oldDMwLT")
+            tau.idMVArun2NewDM = id6(tau, "by%sIsolationMVArun2v1DBnewDMwLT")
             tau.idCI3hit = id3(tau, "by%sCombinedIsolationDeltaBetaCorr3Hits")
+#            tau.idCI3hitdR03 = id3(tau, "by%sCombinedIsolationDeltaBetaCorr3HitsdR03")
             tau.idMVAOldDMRun2 = id3(tau, "by%sIsolationMVArun2v1DBoldDMwLT")
             tau.idMVAOldDMRun2dR03 = id3(tau, "by%sIsolationMVArun2v1DBdR03oldDMwLT")
```

Comment:
This is the same discriminator as in line 85, but here not all 6 working points are saved, so I'd suggest to remove it here.